waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -782,12 +782,20 @@
    :periodSeconds health-check-interval-secs
    :timeoutSeconds 1})
 
+(defn envoy-sidecar-enabled?
+  "Returns true if the envoy sidecar is enabled, and false otherwise"
+  [_ _ {:strs [env]} _]
+  (let [reverse-proxy-flag "REVERSE_PROXY"]
+    (if (contains? env reverse-proxy-flag)
```

Comment:
Nitpick: `if` is unnecessary, the `contains?` will return true/false.

Reply:
Resolved